src/eval-frame/components/reps/tiny-reps.jsx

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import styled from "react-emotion";
+
+const numberColor = "rgb(28, 0, 207)";
+const boolColor = numberColor;
+const nullUndefinedColor = "#808080";
+const classInfoColor = "#808080";
+const stringColor = "rgb(196, 26, 22)";
+const separatorColor = "black";
+const dateColor = stringColor;
+const symbolColor = stringColor;
+const errorColor = "";
+const functionColor = "rgb(28, 30, 207)";
+const ellipsisColor = "#111 ";
+
+const RepBaseStyledSpan = color => styled("span")`
+  color: ${color};
+  font-family: Menlo, monospace;
+  font-size: 12px;
+  line-height: 14px;
+`;
+
+const tinyStringValueWithColor = color => {
+  const ValueSpan = RepBaseStyledSpan(color);
+  return x => <ValueSpan>{x.stringValue}</ValueSpan>;
+};
+
+const ClassInfoSpan = RepBaseStyledSpan(classInfoColor);
+
+const objectRep = obj => (
+  <ClassInfoSpan>
+    {obj.class}({obj.size})
```

Comment:
something to consider playing with:

(1) moving to comma int, so that we're not stuck with `100000` but instead `100,000` will make it easier to read.
(2) moving the parens to `<sub>{obj.size}</sub>` would make the total volume subscripted and less of a big wall of stuff
(3) if you don't like (2) (I'm not even sure how I feel about it) perhaps reduze the text weight for `obj.size` to `300` or something smaller than `normal` so it is a bit easier to parse. The monospace rendering as-is can get tough on the eyes when there are a lot of values in the object.

this might be the wrong approach, but this was something I noticed while looking through the storybook

Reply:
interesting. i don't have strong opinion about any of these suggestions, and none of them strike me as "bad", but i guess i'm initially inclined to follow the example from chrome and FF devtools until we've dogfooded enough to see where pain points show up, at which point we can refine? does that sound ok?

Reply:
sounds sensible :)